### PR TITLE
make decoys buckleable

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
title

## Why / Balance
pour one out for a syndie homie, now for the whole squad

## Technical details
fake body <ins>/!\\</ins>

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- tweak: Decoy balloons can now sit on chairs.

### CODE REVIEW

File: Resources/Prototypes/Body/Prototypes/decoy.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿- type: body
+  id: Decoy
+  name: decoy
+  root: root
+  # fake part
+  slots:
+    root:
+      part: null
```
Comment (metalgearsloth) [inline Resources/Prototypes/Body/Prototypes/decoy.yml:8]: Is this for buckle coz looks sus af.

Comment (deltanedas) [reply Resources/Prototypes/Body/Prototypes/decoy.yml:8]: yes its sus af
having no root part crashes client, not having body means it cant be buckled by verb or dragging

**Comment by K-Dynamic** [comment]
The decoys really should spawn with equipped fake bags, fake weapons to complete their look

Buckling them to a chair is good step in making them believable

File: Resources/Prototypes/Body/Prototypes/decoy.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Body/Prototypes/decoy.yml:8]: You should fix that rather than doing dodgy workarounds.

**Comment by mirrorcult** [comment]
dont think this is worth it with the weird asf body workarounds required for this, reopen if you use a different approach

**Pull request closed without merging**